clickhouse/client.go

```diff
@@ -0,0 +1,167 @@
+package clickhouse
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"time"
+
+	"github.com/ClickHouse/clickhouse-go/v2"
+	"github.com/ClickHouse/clickhouse-go/v2/lib/driver"
+	"github.com/chainbound/valtrack/log"
+	"github.com/rs/zerolog"
+)
+
+func ValidatorMetadataDDL(db string) string {
+	return fmt.Sprintf(`CREATE TABLE IF NOT EXISTS %s.validator_metadata (
```

Comment:
This should be the table in sqlite, not Clickhouse. Clickhouse only stores the regular events

Reply:
Made the necessary changes! 
- The `ValidatorEvent` is published to clickhouse
- The validator tracker table is stored on the local sqlite database named `validator_tracker.sqlite`